tests/features/eap/6.4/eap.feature

```diff
@@ -1,26 +1,25 @@
-@jboss-eap-6/eap64-openshift
+
 Feature: Openshift EAP basic tests
 
-  @ci
-  Scenario: Check that the jboss-eap-6/eap64-openshift image contains 6 layers
+  Scenario: Check that the  image contains 6 layers
```

Comment:
This doesn't seem correct

Reply:
Yep, thanks for spotting these, will update.